### ISSUE TITLE
It should be explained whether or not .html() unbinds elements  

### ISSUE THREAD

**Issue by lahabana** [jquery/api.jquery.com#69]
There's nothing said about that in the doc.
That matter is documented for both .remove():

> In addition to the elements themselves, all bound events and jQuery data associated with the elements are removed. To remove the elements without removing data and events, use .detach() instead.

and .empty():

> To avoid memory leaks, jQuery removes other constructs such as data and event handlers from the child elements before removing the elements themselves.

It seems to do unbind but I'm not actually sure...
